Git Testing/Scenes/Listing/ListingVC.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ViewController.swift
+//  Git Testing
+//
+//  Created by BrainX Technologies on 14/01/2021.
+//
+
+import UIKit
+
+class ListingVC: UIViewController {
+    
+    //MARK:- IBOUTLETS
+    @IBOutlet var tableView:UITableView!
+    
+    //MARK:- PROPERTIES
+    var viewModel: ListViewModel!
```

Comment:
ViewModel is unnused no purpose of it.
And the app may crash with this implementation if the fetch data api returns failure and you try to use the view model object somehwhere.

Reply:
need to change to optional?